### PR TITLE
fix: DEV-2964: Wrap Annotation History with an interface toggle

### PR DESCRIPTION
**Pull request by nicholasrq**

### CODE REVIEW

File: src/components/SidePanels/DetailsPanel/DetailsPanel.tsx [hunk 1]
```diff
@@ -40,16 +41,25 @@ const Content: FC<any> = observer(({
...
+  const hideHistoryTitle = !historyEnabled && draftsEnabled;
 
   return (
     <>
       <Elem name="section">
-        <Elem name="section-head">
-          Annotation History
-          <span>#{currentEntity.pk ?? currentEntity.id}</span>
-        </Elem>
+        {hideHistoryTitle === false && (
```
Comment (bmartel) [inline src/components/SidePanels/DetailsPanel/DetailsPanel.tsx:51]: Is there a need for the explicit false check, or could this be shortened to `!hideHistoryTitle` ?

Comment (nicholasrq) [reply src/components/SidePanels/DetailsPanel/DetailsPanel.tsx:51]: not really, just used to avoid `!var` things. can't even remember why. lm fix that

**Review by bmartel — APPROVED** [review]

File: src/components/SidePanels/DetailsPanel/DetailsPanel.tsx [hunk 2]
```diff
@@ -41,14 +42,16 @@ const Content: FC<any> = observer(({
...
-  const draftsEnabled = isFF(FF_DEV_2290);
-  const hideHistoryTitle = !historyEnabled && draftsEnabled;
+  const showAnnotationHistory = store.hasInterface("annotations:history");
+  const sohwDraftInHistory = isFF(FF_DEV_2290);
 
   return (
     <>
       <Elem name="section">
-        {!hideHistoryTitle && (
+        {!sohwDraftInHistory && (
```
Comment (juliosgarbi) [inline src/components/SidePanels/DetailsPanel/DetailsPanel.tsx:51]: There is a typo here, it should be showDraftInHistory I think.

File: src/components/SidePanels/DetailsPanel/DetailsPanel.tsx [hunk 3]
```diff
@@ -57,8 +60,8 @@ const GeneralPanel: FC<any> = inject("store")(observer(({ store, currentEntity }
         <Elem name="section-content">
           <AnnotationHistory
             inline
-            showDraft={draftsEnabled}
-            enabled={historyEnabled}
+            showDraft={sohwDraftInHistory}
```
Comment (juliosgarbi) [inline src/components/SidePanels/DetailsPanel/DetailsPanel.tsx:63]: Typo

**Review by juliosgarbi — CHANGES_REQUESTED** [review]

**Review by juliosgarbi — APPROVED** [review]
LGTM

**Review by hlomzik — APPROVED** [review]

**Review by hlomzik — APPROVED** [review]

**Pull request merged**